Bottom: cb1114478940cb1b0d7fabdc73f72ccae08b29e6
Top:    5978fd8303959184c79386b4929214d93e855a74
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-21 19:25:37 +0000

Refresh of add_pours

---

diff --git a/src/draw.c b/src/draw.c
index 113cf81..29846a0 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1769,7 +1769,7 @@ DrawPolygonLowLevel (PolygonTypePtr Polygon)
  * lowlevel drawing routine for pours
  */
 static void
-DrawPourLowLevel (PourTypePtr Pour, void *data)
+DrawPourLowLevel (PourTypePtr Pour)
 {
   int *x, *y, n, i;
 
@@ -2246,7 +2246,7 @@ DrawPlainPolygon (LayerTypePtr Layer, PolygonTypePtr Polygon)
   if (gui->thindraw_pcb_polygon != NULL &&
       (TEST_FLAG (THINDRAWFLAG, PCB) ||
        TEST_FLAG (THINDRAWPOLYFLAG, PCB) ||
-       TEST_FLAG (CLEARLINEFLAG, Polygon))) {
+       TEST_FLAG (CLEARLINEFLAG, Polygon)))
     gui->thindraw_pcb_polygon (Output.fgGC, Polygon, clip_box);
   else
     gui->fill_pcb_polygon (Output.fgGC, Polygon, clip_box);
